packages/block-library/src/quote/transforms.js

```diff
@@ -1,8 +1,27 @@
 /**
  * WordPress dependencies
  */
-import { createBlock } from '@wordpress/blocks';
-import { create, join, split, toHTMLString } from '@wordpress/rich-text';
+import {
+	createBlock,
+	parseWithAttributeSchema,
+	serialize,
+	switchToBlockType,
+} from '@wordpress/blocks';
+
+const toBlocksOfType = ( blocks, type ) => {
+	const result = [];
+	blocks.forEach( ( block ) => {
+		if ( type === block.name ) {
+			result.push( block );
+		} else {
+			const newBlocks = switchToBlockType( block, type );
+			if ( newBlocks ) {
+				result.push( ...newBlocks );
+			}
+		}
+	} );
+	return result.filter( Boolean );
```

Comment:
this is more of a nitpick, so feel free to disregard, but would it make more sense to filter earlier than at the end? possibly by filtering `newBlocks` where all of the individual lists are tiny?

```ts
result.push( ...(newBlocks.filter(Boolean)) )
```